config/pipeline.yaml

```diff
@@ -304,6 +305,16 @@ platforms:
     mach: amlogic
     dtb: dtbs/amlogic/meson-g12b-a311d-khadas-vim3.dtb
 
+  mt8192-asurada-spherion-r0:
```

Comment:
This will likely be a problem: this platform is already defined in `platforms-chromeos.yaml`, and all YAML files are merged into a single config object, meaning we could end up with an undefined behaviour here depending on which file is parsed first...

Technically this PR could be reduced to the added platform line in the scheduler entry, but OTOH I'd love to keep `pipeline.yaml` self-contained, in the sense that it shouldn't reference a platform defined in a separate file.

For now the best way to be to add an entry in `scheduler-chromeos.yaml` for executing the needed job on spherion (no need for an additional job entry, as we assume `pipeline.yaml` is always available, and therefore can reference jobs defined here in other files).

Also, the `-chromeos` naming of those config files may be confusing, it refers to "devices meant to be running ChromeOS but that can run any mainline Linux-based system", not "ChromeOS-specific tests and jobs".

Reply:
Ah ok, I thought the chromeos files were an overlay for the chromeos instance, like in the old system. I'll just add the test entry in the chromeos file like you mentioned then.

Reply:
@a-wai on second thought, we need the `nfsroot` to be set to the kselftest rootfs, but the base device definition in `arm64-chromebook-device:` also sets it for the chromebooks. Is there a guarantee in the order that `params` are processed? Can we set `nfsroot` to the kselftest rootfs in the job or the scheduler definition and have it always override the device one?

Reply:
Hmm, it's actually the [other way around](https://github.com/kernelci/kernelci-core/blob/main/kernelci/runtime/__init__.py#L152): platform params override job params... We could probably change that, but it needs a bit more thinking...

Reply:
Sounds reasonable to me. The only issue would be if we ever want to override a param for a specific platform without needing to define it for every other platform, but I also can't think of an example right now, so maybe it won't ever be an issue.

Anyway, I've updated the PR now like you suggested, and took the opportunity to enable the test for all MTK Chromebooks instead of just Spherion. So as soon as the change in param overriding order is done this should be ready to go. Are you going to look into that or should I? @a-wai

Reply:
Done as https://github.com/kernelci/kernelci-core/pull/2465

Reply:
Thank you!